zerver/lib/users.py

```diff
@@ -449,6 +457,7 @@ def get_raw_user_data(realm: Realm, acting_user: UserProfile, client_gravatar: b
                                             acting_user = acting_user,
                                             row=row,
                                             client_gravatar= client_gravatar,
-                                            custom_profile_field_data = custom_profile_field_data,
+                                            exclude_old_users_avatars=exclude_old_users_avatars,
+                                            custom_profile_field_data = custom_profile_field_data
```

Comment:
The whitespace is weird/inconsistent here, and also you should have a trailing comma here.